### ISSUE TITLE
dialyzer doesn't work in APPS_DIR setup

### ISSUE THREAD

**Issue by goertzenator** [ninenines/erlang.mk#404]
I have a project with a number of local applications in the `apps/` dir.  `make dialyze` fails as per below:

```
g550jk embnode # make dialyze V=1
dialyzer --no_native --src -r src -Werror_handling -Wrace_conditions -Wunmatched_returns 
  Checking whether the PLT /device/apps/embnode/.embnode.plt is up-to-date... yes
  Proceeding with analysis...
dialyzer: Analysis failed with error:
No .erl files to analyze

erlang.mk:5788: recipe for target 'dialyze' failed
make: *** [dialyze] Error 1
```

The [default dialyze directory](https://github.com/ninenines/erlang.mk/blob/master/plugins/dialyzer.mk#L12) does not look into the `apps/` directory.  As a work-around I added `DIALYZER_DIRS = --src -r apps` to my top level Makefile and then my apps appeared to dialyze correctly.

I think that `make dialyze` should dialyze all the `APPS_DIR` apps by default.

**Comment by essen** [comment]
Yeah this one needs fixing quickly. Thanks for the report.

**Comment by essen** [comment]
This particular issue is fixed, however #405 is still an issue. Closing, thanks!

**Comment by goertzenator** [comment]
Works for me, thanks!
